crates/matrix-sdk-common/src/deserialized_responses.rs

```diff
@@ -166,24 +167,28 @@ impl VerificationState {
 
 /// The sub-enum containing detailed information on why a message is considered
 /// to be unverified.
-#[derive(Clone, Debug, Deserialize, Serialize, PartialEq, Eq)]
+#[derive(Clone, Debug, Deserialize, Error, Serialize, PartialEq, Eq)]
```

Comment:
Yes, the `Error`-ness itself isn't used, but it does allow me to define the `Display` trait by using the `#[error(..)]` attribute so that `[MegolmError](SenderIdentityNotTrusted)`.  It's slightly abusing the macro, but I like the `#[error(..)]` attribute because it seems cleaner to me than creating an `impl` block for it.  Alternatively, I could use something like the [`derive_more`](https://docs.rs/derive_more/latest/derive_more/derive.Display.html) crate, though I'm not a fan of adding extra dependencies.

Reply:
if all you need is an `impl Display`, can't you just write one? Should be pretty simple.